docs/data/material/getting-started/templates/templates.md

```diff
@@ -5,7 +5,7 @@ title: 9+ Free React Templates
 
 # React Templates
 
-<p class="description">A selection of free react templates to help you get started building your app. The collection contains react dashboard, react admin, and more.</p>
+<p class="description">A selection of free Material UI templates to help you get started building your app. The collection contains react dashboard, react admin, and more.</p>
```

Comment:
I'm unsure with "react dashboard" and "reach admin" here refer to an actual package, thus the lowercase, or if we should capitalize them to React dashboard and React admin instead. 🤔

Reply:
Exactly! Well, I'm happy to not touch it at this moment but thought of just giving a heads-up. Handing it to you 😬